### PR TITLE
Cukinia tests ci

### PR DESCRIPTION
**Pull request by eroussy**
This github action ensure that the commit of the pull request have been
tested of the seapath/ansible repository.

### CODE REVIEW

File: .github/workflows/certify-commit.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  certify-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - run : git clone https://github.com/seapath/ansible.git;
```
Comment (deribaucourt) [inline .github/workflows/certify-commit.yml]: `git clone -b debian-main` to avoid git switch

File: .github/workflows/certify-commit.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+              cd ansible;
+              git switch debian-main;
+              git submodule update --init -f;
+              cd src/cukinia-tests/;
+              git log --format=format:%H | grep $GITHUB_SHA;
```
Comment (deribaucourt) [inline .github/workflows/certify-commit.yml:20]: Did you test this?

**Review by deribaucourt — CHANGES_REQUESTED** [review]

File: .github/workflows/certify-commit.yml (hunk 2 above)
Comment (eroussy) [reply .github/workflows/certify-commit.yml:20]: yes, i test it on my fork with hardcoded GITHUB_SHA.
I don't see any ways to test it directly

**Review by dupremathieu — COMMENTED** [review]
Is Merging the ansible bump submodule commit before is acceptable?

**Comment by eroussy** [comment]
> Is Merging the ansible bump submodule commit before is acceptable?

Yes
On the Ansible repository, the submodule cukinia-tests contains the hash of the commit and not a revision like main.
So even if we want to update only cukinia-test, we have to make a PR on Ansible ( and this PR contains only the bump )
Merging Ansible before cukinia-tests ensure that the CI was successful.

The only remaining problem could be if we forget to merge the cukinia-tests PR, but I see no way to check that.

**Review by deribaucourt — APPROVED** [review]
LGTM. Will prevent accidental merges, thanks!
As a nice to one, log that we would like to deploy the cukinia test and directly test it in a future version of the CI.

**Pull request closed without merging**